src/worker.js

```diff
@@ -33,12 +33,11 @@ function lintJob(argv, contents, eslint, configPath, config) {
 }
 
 function fixJob(argv, eslint) {
-  try {
-    eslint.execute(argv)
-    return 'Linter-ESLint: Fix Complete'
-  } catch (err) {
-    throw new Error('Linter-ESLint: Fix Attempt Completed, Linting Errors Remain')
+  const exit = eslint.execute(argv)
+  if (exit === 0) {
+    return 'Linter-ESLint: Fix complete.'
   }
+  return 'Linter-ESLint: Fix attempt complete, but linting errors remain.'
```

Comment:
Actually, to get an error code other than `0`, means there are errors.  If there are only warnings, eslint will exit with a `0`.